published/external/afxdp.h

```diff
@@ -16,6 +17,11 @@
 extern "C" {
 #endif
 
+
+#if defined(_KERNEL_MODE)
+typedef VOID XDP_API_PROVIDER_BINDING_CONTEXT;
```

Comment:
this seems to be a duplicate definition with xdpapi.h.

also, should this be a DECLARE_HANDLE?

Reply:
what file name is appropriate for this XDP_API_PROVIDER_BINDING_CONTEXT ? and where should I place?

Reply:
Are there any existing headers in common for xdpapi.h and afxdp.h? If not, does it really matter if just this line is duplicated?

Reply:
>Are there any existing headers in common for xdpapi.h and afxdp.h? If not, does it really matter if just this line is duplicated?

We should not define the same type in two different files intended for inclusion in the same type of code. We should pick one or the other (if an implicit inclusion already occurs) or define the type in a new file under `xdp/` and include it in both.

> what file name is appropriate for this XDP_API_PROVIDER_BINDING_CONTEXT ? and where should I place?

I think `xdpapi.h` would be most logical, if you can get it to work. Otherwise, a new header under `xdp/` would be best.

Reply:
doing `xdp/` direction. defining in xdpapi.h only is much more harder than expected